src/material/core/option/option.scss

```diff
@@ -84,7 +79,7 @@
     // MatOption uses a child `<div>` element for its focus state to align with how ListItem does
     // its focus state.
     .mat-mdc-option-pseudo-checkbox, .mdc-list-item__primary-text, > mat-icon {
-      opacity: mdc-list-variables.$content-disabled-opacity;
+      opacity: 0.38;
```

Comment:
This feels like maybe it should come from a token

Reply:
For now I'm focusing on forking off the styles, but once it's done, it should be easy to tokenize it.